[WIP] Prepare 0.1.0 release

itziakos opened the pull request:
Changes
---------

- [ ] Update changelog.
- [ ] Update readme with test and documentation build info.
- [ ] Minor cleanups.
- [ ] bump to version 0.1.0
- [ ] bump to version 0.1.1.dev1

virtuald commented on CHANGELOG.txt:
```diff
@@ -4,6 +4,13 @@ Change Log
+-------------
+
+- Fix: Python 3.5 and 3.6 support (#52).
+- API additions to allow pywin32-ctyps to work with pyinstaller (#46
```
s/ctyps/ctypes/

virtuald commented:
I discovered an incompatibility in the APIs we introduced. I'll have a PR for you shortly.

virtuald commented:
Also, thanks for implementing the unit tests -- I think I have a fix for those also.

virtuald commented:
It looks like the pyinstaller tests pass using pywin32-ctypes without msys2, so I have no further changes to suggest here and am looking forward to the impending release. 👍

itziakos commented:
Thanks @virtuald there are still some issues with UpdateResource which I am working out in #58. As soon as that is fixed/merged, I will update this PR merge and make a release.

itziakos commented:
@virtuald  I have updated the UpdateResource code to better follow the pywin32 behavior (e.g. UpdateResource does not have a return value) and added a few more tests. Please have another look to make sure that these changes work for your usecase.

virtuald commented:
@itziakos still works here :shipit:

itziakos commented:
https://pypi.python.org/pypi/pywin32-ctypes/0.1.0
